src/views/EditFolder/EditFolder.tsx

```diff
@@ -25,13 +27,15 @@ interface OwnProps {
 
 interface MappedProps {
     folderName: string;
+    folderFail: string;
```

Comment:
Maybe it is better to call it something like errorMessage?

Reply:
I think that I did like your style)